### DISCUSSION TITLE
Error in wfb-cli

### DISCUSSION THREAD

**Discussion by SzateX** [svpcom/wfb-ng#141] [Questions]
Hi. It seems I have ok connection by wifibroadcast, but when i'm sending RTP packets over wifibroadcast, and run wfb-cli I got:
`addwstr() returned ERR` in video section of wfb-cli running on my gs.

Could you solve my problem?

**Comment by svpcom** [comment]
Try to resize your terminal window. Ncurses return an error if text is out of screen
